tests/integration_tests/reports/scheduler_tests.py

```diff
@@ -35,150 +35,144 @@ def owners(get_user) -> list[User]:
     return [get_user("admin")]
 
 
-@pytest.mark.usefixtures("owners")
```

Comment:
No need to both specify the use of the fixture here in addition to passing it in to the function.